src/frequenz/client/microgrid/_metadata.py

```diff
@@ -25,22 +22,9 @@ class Location:
     timezone: ZoneInfo | None = None
     """The timezone of the microgrid.
 
-    If not passed during construction (or `None` is passed), and there is a `longitude`
-    and `latitude`, then the timezone wil be looked up in a database based on the
-    coordinates. This lookup could fail, in which case the timezone will still be
-    `None`.
+    If not provided during construction, the timezone will remain `None`.
     """
```

Comment:
Replace with something like:


```suggestion
"""The timezone of the microgrid"""
```